app/native/components/HelloWorld.js

```diff
@@ -1,5 +1,5 @@
-import React, {
-  Component,
+import React, { Component } from 'react';
+import {
```

Comment:
Proptypes is not part of react native anymore, you can move it beside the component import.